api/controllers/transactions.js

```diff
@@ -46,6 +46,10 @@ function transactionFormatter(transaction) {
 	result.recipientPublicKey = result.recipientPublicKey || '';
 	result.signSignature = result.signSignature || '';
 	result.signatures = result.signatures || [];
+	if (transaction.type === 2) {
```

Comment:
can we use the constants like the one below `transactionTypes.DELEGATE` in the code that way we dont have to relay on `2`.

Reply:
Updated code addressing this comment +1